### PR TITLE
Json sample report

### LINKED ISSUE #34: Introduce new action to get sample report as JSON
Z kbc-ui chceme uplne vyhodit pracu s CSV. Momentalne je tu akcia `getSampleReport`, ktora prave vracia CSV.

Mala by pribudnut nova akcia `getSampleReportAsJson` (alebo ina), ktora mi vrati sample data v JSON formate.

### PR DESCRIPTION
**Pull request by ujovlado**
Fixes #34 

- reorganized some output functions (separated to allow usage in json sample function)
- new function and action to get sample data as json

### CODE REVIEW

File: src/Keboola/GoogleAnalyticsExtractor/Extractor/Extractor.php [hunk 1]
```diff
@@ -171,6 +171,26 @@ public function getSampleReport($query)
         ];
     }
 
+    public function getJsonSampleReport($query)
```
Comment (MiroCillik) [inline src/Keboola/GoogleAnalyticsExtractor/Extractor/Extractor.php]: Len drobnost, ze by mi davalo vacsi zmysel, keby sa to volalo "getSampleReportJson". Teda "Json" by bola pripona k povodnemu nazvu.

File: src/Keboola/GoogleAnalyticsExtractor/Extractor/Output.php [hunk 2]
```diff
@@ -64,16 +62,52 @@ public function createReport($query)
...
+    }
 
+    public function createReport($query)
+    {
+        $csv = $this->createCsvFile(sprintf('%s_%s', $query['outputTable'], uniqid()));
+        $csv->writeRow($this->createHeaderRowFromQuery($query));
         return $csv;
     }
 
+    public function createJsonSampleReport($query, $report)
```
Comment (MiroCillik) [inline src/Keboola/GoogleAnalyticsExtractor/Extractor/Output.php]: A toto by som radsej nazval `createReportJson` alebo tiez `createSampleReportJson` aby to bolo konzistentne v ramci tejto triedy.

**Comment by ujovlado** [comment]
Sorry, este zabudnuty rename, je to fixed.

Na ten co je pre PR som dal cancel, preto nepresiel. A vypol som buildy na PRs, ostali len branches.

**Comment by MiroCillik** [comment]
Dobre, mergol som to rovno.

**Pull request merged**